client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -88,6 +88,7 @@ const FeedbackForm = () => {
                     size="sm"
                     color="white"
                     data-testid="creating-spinner"
+                    className="ticket-create-spinner"
```

Comment:
A fix for the spinner spacing is handled already in this PR https://github.com/TACC/Frontera-Portal/pull/319
```suggestion
```